### DISCUSSION TITLE
Documenting SPSD/M Variable (and parameter?) Names

### DISCUSSION THREAD

**Discussion by alex-rand** [PolicyEngine/policyengine-canada#29] [General]
@MaxGhenis we talked about this earlier this week, but I forgot what the best way to do this is. Like just include the corresponding SPSD/M variable name in a comment at the top of each variable file? I think you also mentioned you wanted to confer with @nikhilwoodruff about something related to this.

**Comment by MaxGhenis** [comment]
Filed https://github.com/PolicyEngine/policyengine-canada/issues/30. For now could you put it in a comment?
